lib/variables.less

```diff
@@ -247,6 +247,77 @@
   margin-top: -1px;
 }
 
+.ltr-linear-gradient(@color) {
+  background: @color;
+  background: -moz-linear-gradient(left, @color 10%, rgba(255, 255, 255, 0) 42%);
+  background: -webkit-linear-gradient(left, @color 10%, rgba(255, 255, 255, 0) 42%);
+  background: -o-linear-gradient(left, @color 10%, rgba(255, 255, 255, 0) 42%);
+  background: -ms-linear-gradient(left, @color 10%, rgba(255, 255, 255, 0) 42%);
+  background: linear-gradient(to right, @color 10%, rgba(255, 255, 255, 0) 42%);
+}
+
+.animation(@animation, @timing, @iteration, @timing-function) {
+  -moz-animation: @animation @timing @iteration @timing-function;
+  -webkit-animation: @animation @timing @iteration @timing-function;
+  animation: @animation @timing @iteration @timing-function;
+}
+
+.loader(@color, @spinner) {
+  font-size: 10px;
+  margin: 50px auto;
+  text-indent: -9999em;
```

Comment:
since we follow the convention of clipping text with `clip: rect(...)`, this `text-indent` is unnecessary